AuctionHouse/Overview.md

```diff
@@ -0,0 +1,101 @@
+# Auction House: Overview
+
+## Introduction
+Auction House is a program that allows users to exchange assets within the Solana blockchain. 
+
+There are plenty of ways to exchange assets on Solana, so why another protocol focused on solving this problem? Let's dive deep into that.
+
+The ethos of this program is to allow anyone to create and configure their own marketplace and even provide their own custom logic on how assets should be exchanged. The motivation behind the Auction House protocol is to create a healthy ecosystem of marketplaces that focus on different use-cases, and more importantly, each bringing their own flavor into the way they allow users to trade assets.
+
+The most important aspect of the Auction House protocol is that it provides ownership of assets to the user.
+
+Traditionally, as soon as the user lists an asset on a marketplace, the asset is moved from the user's wallet into a wallet known as the [Escrow](https://www.investopedia.com/terms/e/escrow.asp) wallet owned by the marketplace and is kept or **escrowed** there until the asset is delisted or sold. This poses some concerns:
+* The user can not list the same asset on multiple marketplaces
+* The user has to rely on the marketplaces’ escrow contracts to safely hold their asset.
+
+This is where Auction House shows its power. Its a transaction protocol that allows marketplaces to implement an **escrow-less** sales contract, thus providing ownership of assets to the users.
+
+
+## Creating an Auction House
+The Auction House program can be used to make a marketplace by creating a new **Auction House Instance**. The `Auction House Instance` is a [Program Derived Address (PDA)](https://solanacookbook.com/core-concepts/pdas.html) which is in the keyspace of the Auction House program, ie, the `Instances` are simply public keys owned by the Auction House program.
+
+![](https://i.imgur.com/2HPpM9g.png =300x)
+
+The Instance can be configured in the whatever way the user wants. Here are some important configurable parameters:
+
+* `requireSignOff`: this allows marketplaces to gate which asset can be listed, bought and sold. For every action: list, bid, sale, the instance [authority](https://docs.solana.com/staking/stake-accounts#understanding-account-authorities) needs to sign the transaction
```

Comment:
- "bought and sold" is confusing because they are synonyms outside of this context.
 
```suggestion
* `requireSignOff`: this allows marketplaces to gate which assets can be listed and which bids can be placed. On every relevant instruction, the Auction House [authority](https://docs.solana.com/staking/stake-accounts#understanding-account-authorities) needs to sign the transaction.
```